common/supplemental/ordinals.xml

```diff
@@ -35,6 +35,10 @@ CLDR data files are interpreted according to the LDML specification (http://unic
             <pluralRule count="one">n = 1..4 @integer 1~4</pluralRule>
             <pluralRule count="other"> @integer 0, 5~19, 100, 1000, 10000, 100000, 1000000, …</pluralRule>
         </pluralRules>
+        <pluralRules locales="es">
+            <pluralRule count="one">n % 10 = 1,3 @integer 1, 3, 11, 13, 21, 23, 31, 33, 41, 43, 51, 53, 101, 103, 1001, 1003, …</pluralRule>
```

Comment:
You forgot the ` and n % 100 != 11 and  and n % 100 != 13`. This rule doesn’t apply to 11 and 13 as can be seen in the [Number Format Tester](https://st.unicode.org/cldr-apps/numbers.jsp?locale=es&type=OrdinalRules).

It’s worth mentioning that these rules only apply towards the masculine adjective form of ordinals and not the other ordinal forms.

Reply:
Hi @grhoten, Thanks for the review,

**For orindal 13:**
I believe the [Number Format Tester](https://st.unicode.org/cldr-apps/numbers.jsp?locale=es&type=OrdinalRules) could be incorrect. You can check the `spellout-ordinal-masculine-adjective` in [the rbnf of es.xml](https://github.com/unicode-org/cldr/blob/a643db9ff3195c622a5db561985b2d73c7c586bb/common/rbnf/es.xml#L159-L161). 

```xml
<rbnfrule value="13">decimo→→;</rbnfrule>
```

The masculine adjective ordinal of `13(decimotercero)` will become `decimotercer`. This means it will apply the rule, drop `o` and becomes `13ᵉʳ`


**For orindal 11:**
Meanwhile, the masculine adjective ordinal of `11` is more complex, it could be `undécimo`,`decimoprimero` or `onceno` depends on users in different countries or regions. The rule will only apply to `decimoprimero`, but not to the others.  Reference: [undécimo y decimoprimero, formas válidas](https://www.fundeu.es/recomendacion/decimoprimero-undecimo/)

While [the rbnf of es.xml](https://github.com/unicode-org/cldr/blob/a643db9ff3195c622a5db561985b2d73c7c586bb/common/rbnf/es.xml#L159-L161) prefers to  `undécimo`, I am keeping `11` unchanged and fixing problems on `13` in rnbf and ordinals.xml

Please help to review again.